Client/src/Pages/Settings Pages/StudentSettings/ProfileSettings/StudentProfileSettings.js

```diff
@@ -66,6 +67,42 @@ const ProfileInformation = (props) => {
   const url =
     "https://7jdf878rej.execute-api.us-east-2.amazonaws.com/test/users/students/" +
     id;
+  
+  
+  function displaySaveButton() {
```

Comment:
React renders every time some value is changed. So, you can just use a variable to store whether some value is changed or not. So, currently you are checking whether the button is disabled or not by opacity, you should do it the other way around, i.e., calculate opacity based on whether the button is disabled or active. Opacity styling could change later so that way you won't have to change 0.5 to some other number everywhere later.

Reply:
@ishchhabra changed this functionality and made the code readable